src/layers/regularizers/batch_normalization.cpp

```diff
@@ -208,13 +218,20 @@ void batch_normalization_layer<data_layout::DATA_PARALLEL, El::Device::CPU>::bp_
 
   // Accumulate gradients
   if (is_training) {
-    if (m_use_global_stats) {
+    if (m_stats_aggregation == batch_normalization_stats_aggregation::global) {
```

Comment:
Switch-case statement?

Reply:
I don't think a switch statement when there are precisely two cases is worth while.